tests/utils/execution.hpp

```diff
@@ -30,7 +30,7 @@ template <
     class Test,
     typename validate_func,
     typename KERNEL,
-    int SLMSIZE = 128 * 1024,
+    int SLMSIZE = Test::gpu_arch >= gpu_arch::XeHpc ? 128 * 1024 : 64 * 1024,
```

Comment:
I think there is a field for slm side in arch_config

Reply:
fix